### DISCUSSION TITLE
How to insert paragraph before a code block in text note?

### DISCUSSION THREAD

**Discussion by Zhuang-XiaoJing** [TriliumNext/Trilium#3520] [Q&A]
Pressing enter three times can add paragraph after a code block, how to add paragraph before it ?

**Comment by wunter8** [comment]
Press enter on an empty line at the beginning of the code block
![trilium_60kU32F9CC](https://user-images.githubusercontent.com/8421688/225057269-3f42679e-6aec-4752-942e-0c805261bd1c.gif)
